Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -773,6 +777,102 @@
     <ZipProject Include="ProjectTemplates\AzureExpress4App\setup_web.cmd" />
     <ZipProject Include="ProjectTemplates\AzureExpress4App\Web.config" />
     <ZipProject Include="ProjectTemplates\AzureExpress4App\Web.Debug.config" />
+    <None Include="Snippets\1033\Nodejs\alert.snippet">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>

```

Comment:
All of the snippets must have the IncludeInVSIX property set to true, otherwise they won't be packaged and included in the experimental hive extension folder. Additionally you'll want to set the Build Action to Content rather than None (required to use the IncludeInVSIX property anyways).

Lastly, there's no reason to always copy to the output directory in this scenario - PreserveNewest is sufficient here. Otherwise what happens is that the project will rebuild every time you build.